tests/trainer/connectors/test_checkpoint_connector.py

```diff
@@ -38,6 +39,7 @@ def on_hpc_load(self, checkpoint):
         self.hpc_load_called += 1
 
 
+@pytest.mark.filterwarnings("ignore::DeprecationWarning")
```

Comment:
we would normally wrap the fit call with the `pytest.deprecated_call` ctx manager and match the message